cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -1994,7 +2002,7 @@ Slice::Python::CodeVisitor::collectExceptionMembers(const ExceptionPtr& p, Membe
 }
 
 void
-Slice::Python::CodeVisitor::writeDocstring(const DocCommentPtr& comment, const string& prefix)
+Slice::Python::CodeVisitor::writeDocstring(optional<DocComment> comment, const string& prefix)
```

Comment:
I think these methods should take a reference, as they are not taking ownership of the comment.

Reply:
I changed it to use a `const&`.
I don't think it really matters though since we were always passing in xvalues to this function.